### DISCUSSION TITLE
min fill level then seding, fertalicer.

### DISCUSSION THREAD

**Discussion by raqumax** [Courseplay/courseplay#7306] [Ideas]
Hey. 

First of all this mod make the game so much more enjoyeble. great work guys. 

I just wonder if it is possible to mek e min fille level then seeding and fertilizeing. so then the helper is seeding if you put a min lvl of 5% the helper will not turn ang go in the other direction if the lvl of the implement is lover then this 5% at the end og the field. this will make so you dont run over you new seeded field with your seeding vagon to refill the seeder. 

insted the helper turn the tractor to be ready to drive to the other end of the field and then call for more seed. so you kan refill at the ends insted in the middle of the field. 

the fill level % need to be changeble so you can set it to anything between 1 and 20 % maeby.

**Comment by Tensuko** [comment]
So more an option like, you need at least X % before your start into the next row ? (to make it a bit more simple).

**Reply by rohne83** [reply]
Always thought about something comparable for the combines, which would also prevent starting a new field row in some circumstances, in this case of course if fill level is higher than the setting. So, if in the next field row the combine will have fruit on the pipe side (i.e. every first row of a new land), and the set fill level is reached, it will wait for the unloader before it starts the next row. Or if combine is set to unload itself automatically, it would drive to the trailer to unload before starting a new field row - that way you could place your trailer near one end of the field rows and find an appropriate level so the combine would always unload before it turns there.

**Reply by Tensuko** [reply]
> Always thought about something comparable for the combines, which would also prevent starting a new field row in some circumstances

It does that. We always know when combine will be full, but not as a setting.

**Reply by raqumax** [reply]
> So more an option like, you need at least X % before your start into the next row ? (to make it a bit more simple).

yes precisely
